src/FormField/RegisteredMFAMethodListField.php

```diff
@@ -2,17 +2,41 @@
 
 namespace SilverStripe\MFA\FormField;
 
+use InvalidArgumentException;
 use SilverStripe\Admin\SecurityAdmin;
 use SilverStripe\Forms\FormField;
 use SilverStripe\MFA\Controller\AdminRegistrationController;
 use SilverStripe\MFA\Model\RegisteredMethod;
 use SilverStripe\MFA\Service\MethodRegistry;
 use SilverStripe\MFA\Service\RegisteredMethodManager;
 use SilverStripe\MFA\Service\SchemaGenerator;
+use SilverStripe\Security\Member;
 use SilverStripe\Security\Security;
 
 class RegisteredMFAMethodListField extends FormField
 {
+    public function __construct($name, $title = null, $value = null) {
+        // Because {@see setValue()} can do nothing on e.g. form submission, it is of critical importance that this
+        // object is never in existence without a valid Member set as the value - however as this is a subclass we
+        // cannot alter the constructor function signature to require the parameter.
```

Comment:
This isn't true, constructors aren't bound by the same signature in hierarchy like other PHP methods are

Reply:
Oh 😓